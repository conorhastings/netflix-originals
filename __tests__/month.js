import React from 'react';
import renderer from 'react-test-renderer';
import Month from '../src/components/month.js';


test('Month component renders without error', () => {
  const tree = renderer.create(
    <Month weekRows={[[1,2,3,4,5]]} launches={{}} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});