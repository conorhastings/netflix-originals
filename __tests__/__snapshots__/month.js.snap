// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Month component renders without error 1`] = `
Array [
  <div
    className="css-1a7ck8"
  >
    <div
      className="css-1vew5v7"
    >
      Su
    </div>
    <div
      className="css-1vew5v7"
    >
      Mo
    </div>
    <div
      className="css-1vew5v7"
    >
      Tu
    </div>
    <div
      className="css-1vew5v7"
    >
      We
    </div>
    <div
      className="css-1vew5v7"
    >
      Th
    </div>
    <div
      className="css-1vew5v7"
    >
      Fr
    </div>
    <div
      className="css-1vew5v7"
    >
      Sa
    </div>
  </div>,
  <div
    className="css-121007j"
  >
    <div
      className="css-kmfv35"
    >
      <div
        className="css-1vew5v7"
      >
        <div
          className="css-1i9y2d3"
        >
          1
        </div>
      </div>
      <div
        className="css-1vew5v7"
      >
        <div
          className="css-1i9y2d3"
        >
          2
        </div>
      </div>
      <div
        className="css-1vew5v7"
      >
        <div
          className="css-1i9y2d3"
        >
          3
        </div>
      </div>
      <div
        className="css-1vew5v7"
      >
        <div
          className="css-1i9y2d3"
        >
          4
        </div>
      </div>
      <div
        className="css-1vew5v7"
      >
        <div
          className="css-1i9y2d3"
        >
          5
        </div>
      </div>
    </div>
  </div>,
]
`;
